erpnext/crm/report/prospects_engaged_but_not_converted/prospects_engaged_but_not_converted.py

```diff
@@ -35,22 +35,20 @@ def get_data(filters):
 
 	for lead in frappe.get_all('Lead', fields = ['name', 'lead_name', 'company_name'], filters=lead_filters):
 		data = frappe.db.sql("""
-			select 
-				`tabCommunication`.reference_doctype, `tabCommunication`.reference_name, 
+			select
```

Comment:
Complex query. Inside loop ??  Have  you tried to optimize it.

Reply:
Please refer the following PR [#7263](https://github.com/frappe/frappe/pull/7263)
The query is to make it compatible with new communication structure